src/schedule_synthesis.jl

```diff
@@ -72,16 +72,16 @@ function schedule_xghtc(constraints::Vector{<:MeetAny}; slotsize::Int64=1, H::In
 end
 
 """
-    synthesize_constraints(sysd, K, z_0, d_max, maxwindow, n, t)
+    synthesize_constraints(sysd, K, z_0, d_max, maxwindow, n, H; iterations=ceil(H/n))
 
 Find all `MeetAny` weakly hard constraints with window size at most `maxwindow` that 
 guarantees the deviation upper bound is at most `d_max`. The system is specified by 
-[`Automaton`](@ref) `a` and initial state is `z_0`. `n` and `t` are as in 
-[`bounded_runs_iter`](@ref).
+[`Automaton`](@ref) `a` and initial state is `z_0`. `n` and `iterations` are as `n` 
+and `t` in [`bounded_runs_iter`](@ref).
 """
 function synthesize_constraints(sysd::AbstractStateSpace{<:Discrete},
-    K::AbstractMatrix{Float64}, z_0::AbstractVecOrMat, d_max::Float64,
-    maxwindow::Int64, n::Int64, t::Int64)
+        K::AbstractMatrix{Float64}, z_0::AbstractVecOrMat, d_max::Float64,
+        maxwindow::Int64, n::Int64, H::Int64; iterations::Int64=ceil(H / n))
```

Comment:
Would a user of this function ever actually want to specify `iterations`?  It seems like setting it any smaller than the default would cause a runtime error, and setting it larger would increase the runtime with no change to the result.